Make Buy_page & Sell_page

isoppu1632 opened the pull request:
#WHAT
・購入画面と出品画面のマークアップ

#WHY

Commit:
Merge branch 'master' into Item_Transaction_page

Commit:
Merge branch 'master' into Item_Transaction_page

Commit:
Create Buy_view & Sell_view

#WHAT
・購入画面と出品画面のマークアップ

#WHY

TOMARUMARU commented on app/assets/stylesheets/_item_sell.scss:
```diff
@@ -0,0 +1,125 @@
+    .price ,.profit_price{
+      float: right;
+    }
+    .price_form {
```
ここら辺もBEMと&用いたいですね。他の箇所も同様です

isoppu1632 committed:
Merge branch 'master' into Item_Transaction_page

Commit:
Merge branch 'master' into Item_Transaction_page

Commit:
Fix CSS & Haml

#WHAT
・細かいBEMの設定
・ロゴに画像を使用

#WHY
・BEMにできる部分でBEMを使用していなかった。
・image_tagにURLを使用していた所を画像に変更

TOMARUMARU reviewed (approved):
LGTM

Commit:
Fix Logo

・メルカリのロゴとGoogleのロゴ修正
・sign_in & sign_out 命名変更

isoppu1632 committed:
Merge branch 'master' into Item_Transaction_page
